### ISSUE TITLE
Swagger 2.0

### ISSUE THREAD

**Issue by simong** [oaeproject/Hilary#1148]
This is a pretty minor issue but Swagger's 2.0 spec has been out for a while and it offers some nice things. One thing in particular would be really useful, which is the operation descriptions. It allows you to provide more documentation about an operation and go into more detail. It allows for markdown as well which is kinda nice.

Something like:
```
/**
 * @REST deleteContentContentId
 *
 * Delete a content item
 *
 * Delete a piece of content from the system. The content item will be removed from all the content libraries and/or folders it's part of automatically.
 *
 * @Server      tenant
 * @Method      DELETE
 * @Path        /content/{contentId}
etc..
 */
```

I don't think this needs any changes to the `restjsdoc` module, just some reformatting/restructring in `oae-docs`.

**Comment by stuartf** [comment]
So, the second description is separate from the first one though? And if it can contain markdown I imagine that means it can have line breaks? I think this is doable, though it will take a small change in `restjsdoc`.

**Comment by simong** [comment]
Yes. Although that's just a proposal, I'm more than happy to do it in other (easier?) ways. Swagger now allows you to have a short summary (~100 characters) and a larger blob of notes. Something through which we can add those notes would be really useful

**Comment by stuartf** [comment]
OK, I think we might actually be able to do this without touching `restjsdoc` because everything between the `@REST` and the next `@<tag>` just goes into a description field, so we can split that on the first newline to populate the 2 different descriptions.

**Comment by stuartf** [comment]
Looking at the swagger 2.0 spec it doesn't appear that there are actually 2 description fields, they just changed the `operations` array to a `paths` object, though I think some things like the markdown processing of the description field are new. So we'll need to update the version of swagger-ui on the front end, as well as re-arranging the swagger endpoint's output to match the 2.0 spec.
